### ISSUE TITLE
[enterprise-4.9] Issue in file networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

### ISSUE THREAD

**Issue by Scotchman0** [openshift/openshift-docs#42511]
### Which section(s) is the issue in?

https://docs.openshift.com/container-platform/4.9/networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.html
About subheading

### What needs fixing?

Please include a clarification that the steps to create a new nncp network interface (like  a bridge interface) can only be done on a network interface that is NOT: cluster default, or provisioning interfaces nics. Attempting to utilize the default cluster network interface to create a split bridge connection will succeed to create a new NNCP via the CLI, but during application, the process will auto-revert changes when pushed (since the new interface would disable the primary uplink for the nodes) - it will result in a failing NNCE status with vague reasons as to why - looking in the networkManager journal logs will reveal that the failure was caused by an automatic rollback the moment the network attempts to come online (because doing so will cut network access for the cluster instead of splitting the interface for two separate network tasks. Customers should be aware that this process should not be done on existing/established interfaces like the cluster default interface or the bare-metal interface as applicable. A new NIC should be modified.

Basically if we can add a line that says "do not run this on your nodes' primary interface/cluster-default network" that may prevent headaches moving forward.

**Comment by Scotchman0** [comment]
I see that there is relevant verbiage attached in docs to this effect, thank you!

/close
